lib/suspenders/app_builder.rb

```diff
@@ -1,6 +1,11 @@
+require "forwardable"
+
 module Suspenders
   class AppBuilder < Rails::AppBuilder
     include Suspenders::Actions
+    extend Forwardable
```

Comment:
Do we need `Forwardable`? Does `SimpleDelegator` achieve the same result? How do they differ?